src/pages/[...slug].tsx

```diff
@@ -49,8 +53,10 @@ const layoutMapping = {
   upgrade: UpgradeLayout,
   tutorial: TutorialLayout,
   // event: EventLayout,
-  // docs: DocsLayout,
-} as const
+  docs: DocsLayout,
+}
+
+type LayoutMappingType = typeof layoutMapping
```

Comment:
This in place of `as const` to be utilized in defining the `GetStaticProps` type.